tests/AgDatabaseMove.Unit/StripedBackupSetTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Moq;
+using Xunit;
+
+namespace AgDatabaseMove.Unit
+{
+  public class StripedBackupSetTests
+  {
+
+    [Fact]
+    public void CombinesStripedBackups()
+    {
+      var backupChain = BackupOrder.GetBackupListWithStripes();
+      var stripedBackupSetChain = StripedBackupSet.GetStripedBackupSetChain(backupChain);
+
+      Assert.Equal(backupChain.Count/2, stripedBackupSetChain.Count());
```

Comment:
Instead of `Assert.Equal(backupChain.Count/2`, ..)` I think a better assert would be
`Assert.Equal(backupChain.DistinctBy(...).Count, ...)`

DistinctBy or GroupBy:
https://stackoverflow.com/questions/6231360/linq-query-to-return-distinct-field-values-from-list-of-objects

Reply:
Will take a look

Reply:
Since we have changed the code to use `GroupBy`, having the unit test use `GroupBy` too doesn't make sense anymore.
We won't be checking anything useful.

This unit test will check whether our grouping logic is correct. So I feel we should leave it as is.

I think I some extra checks to make sure each striped backup set is distinct but each backup within a specific striped backup set is the same?

Let me know what you think?

Reply:
`Assert.Equal(backupChain.Count/2, ...)` is a test that is going to fail as soon as records are added or removed from the backup list.
I'd like you to write a test that would continue to pass if another record were added to the backup meta list

Reply:
`Distinct()` works to get the correct count
(Tested by increasing the stripes in each stripedSet)